openedx/core/storage.py

```diff
@@ -4,9 +4,11 @@
 from django.contrib.staticfiles.storage import StaticFilesStorage, CachedFilesMixin
 from pipeline.storage import PipelineMixin, NonPackagingMixin
 from require.storage import OptimizedFilesMixin
+from openedx.core.djangoapps.theming.storage import ComprehensiveThemingAwareMixin
 
 
 class ProductionStorage(
+        ComprehensiveThemingAwareMixin,
```

Comment:
I had no idea that we'd removed these! I'm glad we're adding them back, as this is the way that DB and I designed how comprehensive theming would work.